src/screens/ProfileScreen/elements/TextWithEmoji.tsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
-import {Text, StyleSheet} from 'react-native';
 import emojiRegex from 'emoji-regex';
+import {StyleSheet, Text} from 'react-native';
+
 import {COLORS} from '../../../utils/theme';
+import {LinkableText} from '../../../components/LinkableText';
 import {fonts} from '../../../utils/fonts';
+import {isValidUrl} from '../../../utils/string/StringUtils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been updated and new imports have been added for `LinkableText` and `isValidUrl`. This is a good change as it allows the text to be linkable and validates URLs.
```diff
- import {Text, StyleSheet} from 'react-native';
- import emojiRegex from 'emoji-regex';
- import {COLORS} from '../../../utils/theme';
- import {fonts} from '../../../utils/fonts';
+ import React from 'react';
+ import emojiRegex from 'emoji-regex';
+ import {StyleSheet, Text} from 'react-native';
+ import {COLORS} from '../../../utils/theme';
+ import {LinkableText} from '../../../components/LinkableText';
+ import {fonts} from '../../../utils/fonts';
+ import {isValidUrl} from '../../../utils/string/StringUtils';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->